mgmtsystem_review/i18n/mgmtsystem_review.pot

```diff
@@ -16,255 +16,258 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: mgmtsystem_review
-#: model:ir.module.module,shortdesc:mgmtsystem_review.module_meta_information
-msgid "Management System - Review"
-msgstr "Système de gestion - Revue de direction"
+#: field:mgmtsystem.review.line,action_id:0
+#: selection:mgmtsystem.review.line,type:0
+msgid "Action"
+msgstr "Acción"
```

Comment:
Dont put translated terms in the pot file. Why did you regenerated the pot file ? Please, if you only added a few missing terms to the template, update it manually. Otherwise, it is hard to review.